INFO     conftest:conftest.py:62 App opened without logging in.
INFO     utils.config:test_create_account.py:30 Clicked on 'Allow' button for notification permission.
INFO     utils.config:test_create_account.py:35 کلیک روی دکمه 'ایجاد حساب کاربری'...
INFO     utils.config:test_create_account.py:42 No login page displayed, continuing.
INFO     utils.config:test_create_account.py:46 کلیک روی دکمه 'بازکردن حساب'...
INFO     utils.config:test_create_account.py:49 دکمه 'بازکردن حساب' کلیک شد.
INFO     utils.config:test_create_account.py:53 انتخاب سرور 'UAT'...
INFO     utils.config:test_create_account.py:56 سرور 'UAT' انتخاب شد.
INFO     utils.config:test_create_account.py:60 کلیک روی دکمه 'شروع'...
INFO     utils.config:test_create_account.py:63 دکمه 'شروع' کلیک شد.
INFO     utils.config:test_create_account.py:69 Initial switch status: OFF
INFO     utils.config:test_create_account.py:75 کلیک بر روی تاگل سوییچ موافقت با قوانین و مقررات ...
INFO     utils.config:test_create_account.py:78 Switch status after toggle: ON
INFO     utils.config:test_create_account.py:84 کلیک روی دکمه تایید...
INFO     utils.config:test_create_account.py:86 دکمه تایید کلیک شد و به صفحه وارد کردن شماره تلفن هدایت شد.
INFO     utils.config:test_create_account.py:89 ورود شماره تلفن ...
ERROR    utils.config:test_create_account.py:101 خطا رخ داد: 'function' object is not iterable